lib/hanami/action/configuration.rb

```diff
@@ -84,10 +35,25 @@ def initialize(&blk)
       #   Hanami::Controller.configure do
       #     handle_exception ArgumentError => 400
       #   end
+      setting :handled_exceptions, {}
+
       def handle_exception(exception)
-        @handled_exceptions.merge!(exception)
+        handled_exceptions.merge!(exception)
       end
 
+      # Default Mime type to format mapping
+      #
+      # @since 0.2.0
+      # @api private
+      DEFAULT_FORMATS = {
+        'application/octet-stream' => :all,
+        '*/*'                      => :all,
+        'text/html'                => :html
+      }.freeze
+
+      # TODO: docs
+      setting :formats, DEFAULT_FORMATS.dup
```

Comment:
@timriley `dup` will make it mutable again, was that your intention?

Reply:
@solnic This was actually copied over from the previous configuration class, and yes, I think it's actually the desired outcome, because e.g. the `format` method (just below this setting definition) will add new format entries to this hash.